fsm.cpp

```diff
@@ -20,50 +27,61 @@ int state3_stillness_timer = 0;
 
 void fsm::setupFSM()
 {
-    updateTimersFromFlash();
-    DEBUG_SERIAL_LOG.printf("Timers constants: countdown: %u, duration: %u, stillness: %u\r\n", COUNTDOWN_TIMER, DURATION_TIMER, STILLNESS_TIMER);
-}
-
-bool fsm::updateTimersFromFlash()
-{
-    return api.system.flash.get(COUNTDOWN_TIMER_FLASH_ADDRESS, (uint8_t *)&COUNTDOWN_TIMER, sizeof(COUNTDOWN_TIMER)) &&
+    bool success = api.system.flash.get(COUNTDOWN_TIMER_FLASH_ADDRESS, (uint8_t *)&COUNTDOWN_TIMER, sizeof(COUNTDOWN_TIMER)) &&
            api.system.flash.get(DURATION_TIMER_FLASH_ADDRESS, (uint8_t *)&DURATION_TIMER, sizeof(DURATION_TIMER)) &&
-           api.system.flash.get(STILLNESS_TIMER_FLASH_ADDRESS, (uint8_t *)&STILLNESS_TIMER, sizeof(STILLNESS_TIMER)); 
+           api.system.flash.get(STILLNESS_TIMER_FLASH_ADDRESS, (uint8_t *)&STILLNESS_TIMER, sizeof(STILLNESS_TIMER));
+    if (!success) DEBUG_SERIAL_LOG.println("Error reading flash");
```

Comment:
Please use curly braces around all blocks of code like this if-statement. It is more consistent with the other code and can prevent future mistakes.